README.md

```diff
@@ -68,7 +68,7 @@ Right now, RATS TLS supports the following instance types:
 | Medium      | openssl               | sgx\_la                    | sgx\_la                    | openssl                 |
 | High        | openssl               | sev\_snp                   | sev\_snp                   | openssl                 |
 | High        | openssl               | tdx\_ecdsa                 | tdx\_ecdsa                 | openssl                 |
-| Higher      | openssl               | sgx\_ecdsa                 | sgx\_ecdsa sgx\_ecdsa\_qve | openssl                 |
+| High        | openssl               | sgx\_ecdsa                 | sgx\_ecdsa sgx\_ecdsa\_qve | openssl                 |
```

Comment:
The priority of sgx_ecdsa is 52, tdx\_ecdsa  and sev\_snp  is 42. Should we change the priority of sgx_ecdsa to 42?

Reply:
Good question. It is difficulty to only use 3 levels (low, meduim, high) to define the priority. "Higher" is a quite vague. So does it make sense to write down the real priority number instead of the vague level string here? @zhiminghufighting @YangLiang3

Reply:
sure, let us distinguish them by number priority